app/components/provider_interface/status_box_components/awaiting_provider_decision_component.html.erb

```diff
@@ -1,3 +1,3 @@
 <%= render SummaryListComponent.new(rows: rows) %>
 
-<%= govuk_link_to 'Respond to application', provider_interface_application_choice_respond_path(application_choice), button: true, class: 'govuk-!-margin-bottom-0 govuk-!-display-none-print' %>
+<%= govuk_link_to 'Respond to application', new_provider_interface_application_choice_decision_path(application_choice), button: true, class: 'govuk-!-margin-bottom-0 govuk-!-display-none-print' %>
```

Comment:
I can't see anywhere this component is used anymore, can we delete?

Reply:
Oh nevermind, I see how it's used :)

Reply:
Maybe a bit too much Ruby magic there 🌟